DATA/common/setenv_calib.sh

```diff
@@ -22,7 +22,7 @@ if [[ $BEAMTYPE != "cosmic" ]] || [[ $FORCECALIBRATIONS == 1 ]] ; then
 
     # calibrations for TOF
     if has_detector_calib TOF && has_detector_reco TOF; then
-	if has_detector_matching ITSTPCTOF || has_detector_matching ITSTPCTRDTOF; then
+	if ( has_detectors_reco ITS TPC && has_detector_matching ITSTPCTOF ) || ( has_detectors_reco ITS TPC TRD && has_detector_matching ITSTPCTRDTOF ); then
```

Comment:
Hi @chiarazampolli : just one question: how does the TOF calibration know which traks to assume as input?
ITSTPCTOF or ITSTPCTRDTOF or both?

Reply:
The calib information could in principle be prepared with both. I think that currently we use only ITSTPCTOF, this was my question to @noferini , he already commented to keep it as it is in this PR.